README.md

```diff
@@ -4,10 +4,11 @@
 
 This project consists of documentation, example files, the Tableau Datasource Verification Tool (TDVT) test harness, and a packaging tool that you can use to build and customize a Tableau Connector that uses an ODBC or JDBC driver.
 
-The latest version of the SDK is always targeted towards the latest, non-beta version of Tableau. Right now, this is **Tableau Desktop/Server 2023.3**. This means that the samples may not work on older versions of Tableau, and connectors packaged with newer versions of the SDK may not work in older versions of Tableau. You can download past releases of the SDK to work with older versions of Tableau if necessary.
+The latest version of the SDK is always targeted towards the latest, non-beta version of Tableau. Right now, this is **[Tableau Desktop/Server 2024.1](https://www.tableau.com/support/releases/desktop/2024.1)**. This means that the samples may not work on older versions of Tableau, and connectors packaged with newer versions of the SDK may not work in older versions of Tableau. You can download past releases of the SDK to work with older versions of Tableau if necessary.
 
 | Tool                                             | Latest Version      |
 |--------------------------------------------------|---------------------|
+| [Connector SDK for Tableau 2024.1](https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2024.1)                 | 03-13-2024          |
```

Comment:
Shouldn't this be 2023.3?

Reply:
Yeah, the release should be 2023.3, as the release is capturing a snapshot in case someone needs to go back to an older release. I think this release should be 2023.3, not 2024.1: https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2024.1 (I'm not seeing 2023.3)